src/components/Note/NoteAuthorList/NoteAuthorList.scss

```diff
@@ -0,0 +1,105 @@
+@import "src/constants/style.scss";
+
+.note-author-list {
+  display: flex;
+  flex-direction: row-reverse;
```

Comment:
why not just `row` and have the elements in the desired order inside the .tsx?

Reply:
changed 👍